### ISSUE TITLE
Release 1.3.1

### ISSUE THREAD

**Issue by lrytz** [scala/scala-xml#685]
We backported the stack safe Node.toString to 1.x in https://github.com/scala/scala-xml/pull/684.

Publishing a patch release with that change.

**Comment by lrytz** [comment]
https://github.com/scala/scala-xml/actions/runs/5554367262/jobs/10144068126

> [error] Module scala-xml has an invalid version number: 1.3.1. You must increment the minor version number to publish a source incompatible release.

**Comment by lrytz** [comment]
Not sure where to look. We have `versionPolicyIntention := Compatibility.BinaryCompatible` in the build (https://github.com/scala/scala-xml/blob/1.x/build.sbt#L52). I looked through https://github.com/scalacenter/sbt-version-policy and couldn't see anything obvious. cc @SethTisue

**Comment by SethTisue** [comment]
`build.sbt` has `BinaryCompatible`, so sbt-version-policy believes you ought to be releasing this as 1.4.0.

If you could can it to `BinaryAndSourceCompatible` and the checks pass, then it will allow you to release it as 1.3.1.

But the checks won't pass; if I `set every version := "1.3.1"` and `set every versionPolicyIntention := Compatibility.BinaryAndSourceCompatible` and then `xml/versionPolicyCheck`, it fails with:

```
[error] scala-xml: Failed binary compatibility check against org.scala-lang.modules:scala-xml_2.12:1.3.0! Found 1 potential problems
[error]  * method declaration(java.lang.String,java.lang.String,java.lang.String)Unit in class scala.xml.include.sax.XIncluder does not have a correspondent in other version
[error]    filter with: ProblemFilters.exclude[DirectMissingMethodProblem]("scala.xml.include.sax.XIncluder.declaration")
[error] Module scala-xml:1.3.1 is not source compatible with version 1.3.0. You have to relax your compatibility intention by changing the value of versionPolicyIntention.
```

what the root cause is here, I'm not sure (didn't dig further)

**Comment by SethTisue** [comment]
btw I was extremely puzzled when I was previously in the same situation, as you'll see at https://github.com/scalacenter/sbt-version-policy/issues/126

**Comment by julienrf** [comment]
Do you have suggestions for a better error message?

**Comment by lrytz** [comment]
> what the root cause is of the `declaration` difference, I don't know (didn't dig further)

Probably a mixin forwarder due to a change in JDK > 8. The release runs on JDK 8, so should be fine.

**Comment by lrytz** [comment]
https://github.com/scala/scala-xml/actions/runs/5574179022/jobs/10182443820 went green, the releases are on maven2

**Comment by mkurz** [comment]
Thank you @lrytz!

**Comment by lrytz** [comment]
For the record, 1.3.1 was released for Scala 2.11 / 2.12 / 2.13 JVM, and for Scala 2.12 and 2.13 with Scala.js 1.

With 1.3.0, we also had releases for Scala 2.11 / 2.12 / 2.13 for Scala.js 0.6, but we excluded those here.

**Comment by SethTisue** [comment]
> Do you have suggestions for a better error message?

I'll reply over at https://github.com/scalacenter/sbt-version-policy/issues/126
